docs/3.0.0-beta.x/deployment/amazon-aws.md

```diff
@@ -295,32 +295,32 @@ On your development machine, navigate to your Strapi project root directory:
 npm install pg
 ```
 
-#### 2. Edit the `database.json` file.
+#### 2. Edit the `./config/database.js` file.
 
 Copy/paste the following:
 
-`Path: ./my-project/config/environments/production/database.json`:
-
-```json
-{
-  "defaultConnection": "default",
-  "connections": {
-    "default": {
-      "connector": "bookshelf",
-      "settings": {
-        "client": "postgres",
-        "host": "${process.env.DATABASE_HOST || '127.0.0.1'}",
-        "port": "${process.env.DATABASE_PORT || 27017}",
-        "database": "${process.env.DATABASE_NAME || 'strapi'}",
-        "username": "${process.env.DATABASE_USERNAME || ''}",
-        "password": "${process.env.DATABASE_PASSWORD || ''}"
+`Path: ./my-project/config/database.js`:
+
+```js
+module.exports = ({ env }) => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'bookshelf',
+      settings: {
+        client: 'postgres',
+        host: env('DATABASE_HOST', '127.0.0.1'),
+        port: env.int('DATABASE_PORT', 27017),
```

Comment:
It's a note for all guides AND for the product.
By default, the port we set by default is `27017`.
Even if you are using Postgres, that's not the correct one.

Reply:
This PR add a change so that the default port depends on the database, just a bad copy/paste from me here